src/value.cpp

```diff
@@ -667,7 +668,7 @@ int LangAltValue::read(const std::string& buf) {
     if (lang.front() == '"') {
       lang = lang.substr(1);
 
-      if (lang.empty() || lang.find('"') != lang.length() - 1)
```

Comment:
This is checking that there isn't a `"` character somewhere in the middle of the string. I'm not sure how important that is, but the new code with `lang.back()` changes the logic.

Reply:
Good catch